1.6Funkcje/1.6Funkcje/Program.cs

```diff
@@ -0,0 +1,129 @@
+﻿ChooseTask();
```

Comment:
Generalna uwaga do tego pliku. Używasz rekurencji więc ten program teoretycznie może wyrzucić błąd StackOverflow bo cały czas wchodzisz w głąb. Lepiej byłoby użyć jakiejś pętli.

Reply:
Będę wiedział na przyszłość, ale teraz już tego nie naprawię, bo to by się wiązało z gruntowną zmianą. Chyba że tak trzeba, to mogę zrobić.

Reply:
Nie, jest OK